app/assets/javascripts/submit_form_ajax.js

```diff
@@ -1,7 +1,7 @@
 // Takes in a data object that contains the info to be submitted in the form property: dataString
 //   and the url to submit to the controller
 // Allows data to be submitted from anywhere on the page using Javascript without using the form itself
-function sendFormSubmissionAjax(dataObj, submitTo, responseEl = "") {
+function sendFormSubmissionAjax(dataObj, submitTo, responseEl = "", callback) {
```

Comment:
Are other files in this folder using snake case or camel case to name the files? Can you check once?

Reply:
I can see a mixture of both in the file naming...
<img width="294" alt="Screenshot 2021-02-02 at 19 13 08" src="https://user-images.githubusercontent.com/7622875/106628319-b9adcf00-658a-11eb-8af8-c780e5bcdc57.png">

Reply:
Oh, let's follow one pattern only then. I like `camelCase` but we can go with anyone.
Also this can be a good idea to make many FTO's to change filename, thought?

Reply:
Yes it would be a good FTO... I'll create an issue to discuss whether to go with camel Case or snake Case

Reply:
Broken out into FTO issues here: #9140

Reply:
Awesome 🎉